spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/AdminController.java

```diff
@@ -600,26 +589,38 @@ public ModelAndView gettingStartedWizardForConsumer(Model model) throws IOExcept
         return mav;
     }
 
-    // Refresh the Test Api Key and the URL to call the service in the getting started wizard for consumer
+    // Refresh the Test Api Key and the URL to call the service in the getting
+    // started wizard for consumer
     @RequestMapping(value = "/consumer/gettingstarted/refreshApiKeys/{whichFragmentToLoad}", method = GET)
     public String refreshApiKeys(@PathVariable("whichFragmentToLoad") String whichFragmentToLoad, Model model) {
 
         ServiceApi gettingStartedTestService = loadTestServiceForConsumerWizard();
-        List<ApiKey> apiKeysForTestService = apiKeyRepository.findByLoggedInAPIConsumerAndServiceId(gettingStartedTestService.getId());
+        List<ApiKey> apiKeysForTestService = apiKeyRepository
+                .findByLoggedInAPIConsumerAndServiceId(gettingStartedTestService.getId());
 
-        // Gets the last Getting Started Test Service APIKey, as the Getting Started for Consumer can be used several times (there is size -1 because the array starts in element 0)
+        // Gets the last Getting Started Test Service APIKey, as the Getting Started for
+        // Consumer can be used several times (there is size -1 because the array starts
+        // in element 0)
         ApiKey newApiKey = apiKeysForTestService.get(apiKeysForTestService.size() - 1);
 
         // get the URL(s) for the test service proxy(s)
         Map<String, List<String>> proxyURLPerNewApiKey = new TreeMap<>();
-        List<String> proxiesUrlList = proxyRepository.customSearchForAllProxiesForGivenServiceApiId(newApiKey.getServiceApi().getId())
```

Comment:
These formatting changes make code harder to read, we should keep the formatting from master in this file.